lib/bridge/MatrixHandler.js

```diff
@@ -468,11 +468,53 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
         return;
     }
     else if (cmd === "!quit") {
-        // clientList.filter(
-        //     (bridgedClient) => bridgedClient.server.domain === ircServer.domain
-        // ).forEach((bridgedClient) => {
-        //     bridgedClient.kill();
-        // });
+        // Filter to get the clients for the [specified] server
+        let clients = clientList.filter(
+            (bridgedClient) => bridgedClient.server.domain === ircServer.domain
+        );
+        if (clients.length === 0) {
+            req.log.info(`No bridgedClients for ${event.user_id}`);
+            let notice = new MatrixAction("notice", "You are not connected to any networks.");
+            yield this.ircBridge.sendMatrixAction(adminRoom, botUser, notice, req);
+        }
+        let getRoomsForChannel = (channel) => {
```

Comment:
You're using this in exactly one place. Please in-line this so I don't need to think where else you might call this.